hive/src/main/java/io/confluent/connect/storage/hive/HiveSchemaConverter.java

```diff
@@ -55,6 +68,17 @@
     return columns;
   }
 
+  public static List<FieldSchema>  convertSchemaMaybeLogical(Schema schema) {
```

Comment:
nit: two space between the type and the method name. Surprised that checkstyle doesn't mind